### PR TITLE
mungegithub: Fix flaky issue nagger timings

### PR DESCRIPTION
**Pull request by apelisse**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/1753)

### CODE REVIEW

File: mungegithub/mungers/nag-flake-issues.go [hunk 1]
```diff
@@ -40,7 +40,8 @@ var (
 	timePeriods = map[string]time.Duration{
 		"priority/P0": 2 * 24 * time.Hour,
 		"priority/P1": 4 * 24 * time.Hour,
-		"priority/P2": 7 * 24 * time.Hour,
+		"priority/P2": 2 * 30 * 24 * time.Hour,
+		"priority/P3": time.Duration(1<<63 - 1),
```
Comment (lavalamp) [inline mungegithub/mungers/nag-flake-issues.go]: What do you want to bet me that the comparison code is overflow-safe?

File: mungegithub/mungers/nag-flake-issues.go [hunk 2]
```diff
@@ -30,7 +30,7 @@ import (
 
 const (
 	flakeNagNotifName = "FLAKE-PING"
```
Comment (lavalamp) [inline mungegithub/mungers/nag-flake-issues.go]: sorry for drive by, but can you fix the spelling? (s/Notif/Notify/)

File: mungegithub/mungers/nag-flake-issues.go [hunk 3]
```diff
@@ -30,7 +30,7 @@ import (
...
-	defaultTimePeriod = 14 * 24 * time.Hour
+	defaultTimePeriod = 4 * 24 * time.Hour
```
Comment (lavalamp) [inline mungegithub/mungers/nag-flake-issues.go]: Add a comment: "Treat unprioritized as P1, to get a human to at least prioritize."

**Review by lavalamp — CHANGES_REQUESTED** [review]

File: mungegithub/mungers/nag-flake-issues.go (hunk 2 above)
Comment (apelisse) [inline mungegithub/mungers/nag-flake-issues.go]: Thanks!

File: mungegithub/mungers/nag-flake-issues.go (hunk 3 above)
Comment (apelisse) [inline mungegithub/mungers/nag-flake-issues.go]: Good point

File: mungegithub/mungers/nag-flake-issues.go (hunk 1 above)
Comment (apelisse) [inline mungegithub/mungers/nag-flake-issues.go]: I have a twenty in my pocket, are you going for it?

**Comment by apelisse** [comment]
Updated @lavalamp

File: mungegithub/mungers/nag-flake-issues.go (hunk 1 above)
Comment (lavalamp) [inline mungegithub/mungers/nag-flake-issues.go]: Good thing I said no!

**Review by lavalamp — APPROVED** [review]
/lgtm

**Pull request merged**